spec/datadog/core/configuration_spec.rb

```diff
@@ -485,6 +440,32 @@
       end
     end
 
+    describe '#logger_without_configuration' do
+      subject(:logger_without_configuration) { test_class.send(:logger_without_configuration) }
+
+      it { is_expected.to be_a_kind_of(Datadog::Core::Logger) }
+
+      context 'when configuration is not initialized and DD_TRACE_DEBUG is not set' do
+        it { expect(logger_without_configuration.level).to be ::Logger::INFO }
+
+        it 'returns a logger' do
+          logger_without_configuration
+
+          expect(logger_without_configuration).to be_a_kind_of(Datadog::Core::Logger)
+        end
```

Comment:
1. This is already tested above on line 446 (there is not difference in ENV)
2. It's better to test interface than type, hence by checking `.level` you already confirm that it's an interface you want, hence - this is not needed